set.mm

```diff
@@ -281024,8 +281087,15 @@ the same dimension over the same (nonzero) ring.  (Contributed by AV,
     psraddcl.x $e |- ( ph -> X e. B ) $.
     psraddcl.y $e |- ( ph -> Y e. B ) $.
     $( Closure of the power series addition operation.  (Contributed by Mario
-       Carneiro, 28-Dec-2014.) $)
+       Carneiro, 28-Dec-2014.)  (Proof shortened by SN, 12-Apr-2025.) $)
     psraddcl $p |- ( ph -> ( X .+ Y ) e. B ) $=
```

Comment:
I think this theorem is not needed anymore: it is used in 4 (not OLD) theorems only, so it can be replaced by psraddclg/psraddcld. Credtis should go to psraddclg/psraddcld then.